README.md

```diff
@@ -379,6 +379,8 @@ are not maintained by the `tokio` project. These include:
   pretty printing them.
 - [`spandoc`] provides a proc macro for constructing spans from doc comments
   _inside_ of functions.
+- [`tracing-wasm`] provides a Subscriber / Layer implementation that reports 
+  events and spans via browser `console.log`, and [User Timing API (`window.performance`)].
```

Comment:
nit: I don't think a comma is necessary here:
```suggestion
  events and spans via browser `console.log` and [User Timing API (`window.performance`)].
```

Reply:
Good catch on both accounts. I'll apply both suggestions and include the update to `lib.rs`